src/Components/IndividualReview.jsx

```diff
@@ -63,6 +102,14 @@ const IndividualReview = () => {
           <CommentTab text="There are no comments yet, check back later!" />
         )}
       </section>
+      <section className="Vote-Buttons">
+        <button className="Up-Vote" onClick={upVoteClick}>
+          Up Vote
+        </button>
+        <button className="Down-Vote" onClick={downVoteClick}>
+          Down Vote
+        </button>
```

Comment:
a nit-picky point (and only my opinion so feel free to ignore) but your button being far away from the number it is effecting is a bit confusing - it's nicer to have the change nearby